lib/exercism/user.rb

```diff
@@ -124,11 +124,22 @@ def count_existing_five_a_day
     ActiveRecord::Base.connection.execute(count_existing_five_a_day_sql).field_values("total").first.to_i
   end
 
-
   def five_a_day_exercises
     @exercises_list ||= ActiveRecord::Base.connection.execute(five_a_day_exercises_sql).to_a
   end
 
+  def show_five_suggestions?
+    if onboarded? && five_available?
+      true
+    end
```

Comment:
This is the equivalent of:

```
if true
  true
else
  nil
end
```

This method implementation should be a one-liner:

```
def show_five_suggestions?
  onboarded? && five_available?
end
```

That way the return value will be either true or false, not true or nil.